sites/partners/src/pages/listings/[id]/applications/add.tsx

```diff
@@ -10,33 +11,35 @@ import { useSingleListingData } from "../../../../lib/hooks"
 const NewApplication = () => {
   const router = useRouter()
   const listingId = router.query.id as string
+  const { profile } = useContext(AuthContext)
 
   const { listingDto: listing } = useSingleListingData(listingId)
 
   return (
-    <Layout>
-      <Head>
-        <title>{t("nav.siteTitlePartners")}</title>
-      </Head>
-      <NavigationHeader
-        className="relative"
-        title={t("applications.newApplication")}
-        breadcrumbs={
-          <Breadcrumbs>
-            <BreadcrumbLink href="/">{t("t.listing")}</BreadcrumbLink>
-            <BreadcrumbLink href={`/listings/${listingId}`}>{listing?.name}</BreadcrumbLink>
-            <BreadcrumbLink href={`/listings/${listingId}/applications`}>
-              {t("nav.applications")}
-            </BreadcrumbLink>
-            <BreadcrumbLink href={`/listings/${listingId}/applications/add`} current>
-              {t("t.add")}
-            </BreadcrumbLink>
-          </Breadcrumbs>
-        }
-      />
-
-      <PaperApplicationForm listingId={listingId} />
-    </Layout>
+    !profile?.userRoles?.isLimitedJurisdictionalAdmin && (
```

Comment:
Would prefer a check before the return, like if (!role) return null

Reply:
I did attempt this but it threw errors about having multiple returns. could try again

Reply:
Worked this time!